tools/pipelines/test-real-service-stress.yml

```diff
@@ -100,7 +100,6 @@ stages:
           # Disable colorization for tinylicious logs (not useful when printing to a file)
           logger__colorize: "false" # Need to pass it as string so ADO doesn't convert it into False (capital F) which doesn't work
           logger__morganFormat: tiny
-        additionalSteps:
```

Comment:
The task right beneath this line is the single entry in the additional steps list, so that whole task should go too :) (in both templates).